content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,90 @@
+## Installation
+
+- **Added** a command line [Istio operator](docs/setup/install/operator/) to install and manage the Istio control plane.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly classify ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
+- **Added** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to `MAGLEV` when using the Redis proxy.
+- **Improved** traffic management to direct traffic to the same region and zone.
```

Comment:
Is this for multi cluster env?  Which deployment model?   I assume it is gateway connected MC model only.

Reply:
Multicluster is not needed for this. Please add a link to the docs https://preliminary.istio.io/docs/ops/traffic-management/locality-load-balancing/

Also maybe add `by default` to the end of the sentance

Reply:
accepted.